res/TensorFlowLiteRecipes/Net_Remove_Unnecessary_StridedSlice_000/test.rule

```diff
@@ -0,0 +1,5 @@
+# To check if Unnecessary StridedSlice removed.
+
+RULE    "VERIFY_FILE_FORMAT"      $(verify_file_format) '=' 1
+
+RULE    "STRIDEDSLICE_EXIST"             $(op_count STRIDEDSLICE) '=' 1
```

Comment:
it would better to align `$(verify_file_format)` and `$(op_count STRIDEDSLICE)`

Reply:
It should be `Net_StridedSlice_StridedSlice_000`? (The current test name is `Net_Remove_Unnecessary_StridedSlice_000`)

Reply:
Done